api/MediaTrackConstraints.json

```diff
@@ -203,6 +203,54 @@
           }
         }
       },
+      "cursor": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaTrackConstraints/cursor",
+          "support": {
+            "chrome": {
+              "version_added": "71"
```

Comment:
Chrome only implemented these on `MediaTrackSettings`. I'll find out why.

Reply:
Updated.

Reply:
I don't have an answer, but that shouldn't hold up merging.